docs/LICENSE_OF_DEPENDENCIES.md

```diff
@@ -7,6 +7,7 @@ following works:
 - code.cloudfoundry.org/clock [Apache License 2.0](https://github.com/cloudfoundry/clock/blob/master/LICENSE)
 - collectd.org [ISC License](https://github.com/collectd/go-collectd/blob/master/LICENSE)
 - dario.cat/mergo [BSD 3-Clause "New" or "Revised" License](https://github.com/imdario/mergo/blob/master/LICENSE)
+- github.com/99designs/go-keychain [MIT License](https://github.com/99designs/go-keychain/blob/master/LICENSE)
```

Comment:
Where does this come from?

Reply:
In my local environment, the result of `make check-deps` says the line needs to be added.
Does it not reproduce in your environment?

```
$ go version
go version go1.21.5 darwin/arm64

$ make check-deps
./scripts/check-deps.sh
darwin/amd64

...(snip)...

windows/arm64
--- /var/folders/vh/0l94sp8n6hx8qwhb6wsy_xrh0000gr/T/tmp.d7GAxRLO/LICENSE_OF_DEPENDENCIES.md	2024-01-10 00:45:33
+++ /var/folders/vh/0l94sp8n6hx8qwhb6wsy_xrh0000gr/T/tmp.d7GAxRLO/HEAD	2024-01-10 00:45:33
@@ -4,0 +5 @@
+github.com/99designs/go-keychain


The docs/LICENSE_OF_DEPENDENCIES.md file does not contain the expected entries.

Lines prefixed with '+' should be added to LICENSE_OF_DEPENDENCIES.md and '-'
lines should be removed.

Include a link to the appropriate licenses for any additions.
make: *** [check-deps] Error 1
```

Reply:
Looks weird since that is only an indirect dependency 🤔

Reply:
@Hipska 

After trying with the latest code, the make check-deps error no longer occurred. It seems there was something wrong with my environment. 🙇